### ISSUE TITLE
The *run_network_generator_OpenTopoDEM* tutorial notebook is failing

### ISSUE THREAD

**Issue by mcflugen** [landlab/landlab#1447]
The *run_network_generator_OpenTopoDEM* notebook is failing with the latest version of *rasterio* (*v1.3*). I believe this is due to a regression in *rasterio* and should, ultimately, be fixed there (see rasterio/rasterio#2538 for my attempt at a fix).

Until that fix is made, the offending cell could be modified to be,
```python
x_UTM13, _ = rasterio.transform.xy(transform, [0] * dim_x_transf, np.arange(0, dim_x_transf))
_, y_UTM13 = rasterio.transform.xy(transform, np.arange(0, dim_y_transf), [0] * dim_y_transf)
```
Not pretty but it should work.

**Comment by mcflugen** [comment]
@shelbyahrendt Is this the same issue you were seeing?

**Comment by shelbyahrendt** [comment]
Yes! That was the same problem, but now that I'm looking back I realize there had been a cascade of issues in delineating a watershed later on. I had that cell block running again with a similar workaround:

`x_UTM13, _ = rasterio.transform.xy(transform, np.zeros(dim_x_transf), np.arange(0, dim_x_transf, 1))
_, y_UTM13 = rasterio.transform.xy(transform, np.arange(0, dim_y_transf, 1), np.zeros(dim_y_transf))`

(which looks like it produces the same result.) However, something subtle may have changed in the transformation, because when I try to isolate the largest watershed using the same process of running the FlowAccumulator and get_watershed_masks_with_area_threshold() using the maximum drainage area I no longer get a single watershed (snip below). The previous notebook version used to return a single watershed bounded on the south somewhere around the red line.

![image](https://user-images.githubusercontent.com/36043125/183114480-2ef59853-1a22-42df-958d-e19473b2e21e.png)

My though was that some small change in the coordinate transformation resulted in drainage connectivity between cells on the west or east of the domain (assuming the only changes since creating the notebook were to the rasterio function). Sure enough, there's this strange north-flowing line of cells on the east side of the domain.

![image](https://user-images.githubusercontent.com/36043125/183124977-fabd83a4-7e0f-4743-a325-3795100f1e98.png)

I then tried to strategically adjust the domain bounds that the Topography tool pulls initially to get rid of this region. Due to the shape of the watershed, this didn't work so well so I switched the input lat,lon to a domain centered on Boulder Creek:

![image](https://user-images.githubusercontent.com/36043125/183126201-d65ad268-2ed0-478b-83fb-bbb55806e324.png)

However, the FlowAccumulator still gives funny results; for example, I would have expected the flow to have been routed towards the pink dashed line rather than going along that strange straight block of cells:

![image](https://user-images.githubusercontent.com/36043125/183126736-631b1b02-dd30-471d-903d-e84843f8408c.png)

Have you seen the FlowAccumulator behave like this before?

**Comment by shelbyahrendt** [comment]
code [here](https://github.com/shelbyahrendt/landlab/blob/network_generator_OpenTopo_bug/notebooks/tutorials/network_sediment_transporter/run_network_generator_OpenTopoDEM.ipynb)

**Comment by sebastien-lenard** [comment]
@mcflugen and @gregtucker can we exclude the test on this failing notebook  run_network_generator_OpenTopoDEM? The problem has stood since may-june and prevents the completion of pull requests since then ...

**Comment by mcflugen** [comment]
I've been merging pull requests when this issue is the only error. I had not excluded this notebook from the tests because I thought if we didn't see it failing we might forget about it. Now that we have an open issue for it, maybe that's good enough to remind us.

**Comment by sebastien-lenard** [comment]
Maybe we could make the modification you proposed on the notebook and push it in a pull request?
It may take some time before rasterio push their next version.

**Comment by mcflugen** [comment]
From what @shelbyahrendt says, it seems like there might be a bigger issue so I think I'll just disable testing of this notebook for now and try to fix it in a new pull request.

**Comment by shelbyahrendt** [comment]
@mcflugen I looked into this a bit further, and somehow it seems like the rasterio reprojection to UTM 13 (which used to work as far as I remember) now gives really funny results for the drainage network. (See notebook [here](https://github.com/shelbyahrendt/landlab/blob/network_generator_OpenTopo_bug/notebooks/tutorials/network_sediment_transporter/run_network_generator_OpenTopoDEM.ipynb) showing the FlowAccumulator results on two grids; one in the WGS84 lat/lon output from the Topography tool and one on the UTM13 transformed grid)  

e.g. for the WGS84 the drainage network looks fine:
![image](https://user-images.githubusercontent.com/36043125/183486218-e575083f-e309-4d13-90b2-de52006984a7.png)

but on the re-projected grid there are these funny linear bits that don't match the topography:
![image](https://user-images.githubusercontent.com/36043125/183486321-cb6dfd18-06e1-4e3f-9aab-2b10fc87ca6f.png)

So, I'm guessing the issue is still in the re-projection, although, I'm not totally sure what changed since the rasterio update. We could just get rid of this for now and demo the create_network on the WGS84 lat/lon crs grid; the only catch is that defining a minimum drainage threshold in 'square-degrees' is not super intuitive, but could be a workaround to get the notebook working for now?

**Comment by mdpiper** [comment]
If it helps, I successfully ran this notebook on the *lab* Hub instance, CSDMS kernel, where rasterio 1.2.10 is installed.

**Comment by sebastien-lenard** [comment]
Hi @mcflugen 

how can we process to deactivate the notebook test and let the pull requests going on? Should I make a pull request with the updated landlab/notebooks/exclude.yml file?

**Comment by mcflugen** [comment]
> how can we process to deactivate the notebook test and let the pull requests going on? Should I make a pull request with the updated landlab/notebooks/exclude.yml file?

@sebastien-lenard See #1463
